plat/kvm/x86/multiboot.c

```diff
@@ -71,18 +71,6 @@ void multiboot_entry(struct lcpu *lcpu, struct multiboot_info *mi)
 	if (mi->flags & MULTIBOOT_INFO_CMDLINE) {
 		if (mi->cmdline) {
 			cmdline_len = strlen((const char *)(__uptr)mi->cmdline);
```

Comment:
Actually IMO these are different things: `bi->cmdline_len` reflects what was provided by the boot protocol and it also refers to the length of `bi->cmdline`. If we update `bi->cmdline_len` it would make sense to also update `bi->cmdline`, but I'm not sure there is value on that.

Reply:
Wait sorry somehow I thought you referred to `uk_boot_early_init_parse_cmdline()`. The proposed change makes sense in multiboot, I'll push an update.